Python/Setup/swix/Microsoft.PythonTools.vsmanproj

```diff
@@ -10,13 +10,11 @@
 
     <BuildNumber>$(VSTarget).$(BuildNumber)</BuildNumber>
 
-    <TargetName>Microsoft.PythonTools</TargetName>
-    <OutputPath>$(SetupOutputPath)swix\</OutputPath>
-    <OutDir>$(OutputPath)</OutDir>
+    <TargetName>$(MSBuildProjectName)</TargetName>
   </PropertyGroup>
 
   <ItemGroup>
-    <MergeManifest Include="$(OutputPath)*.json" />
+    <MergeManifest Include="$(OutputPath)Microsoft.*.json" />
```

Comment:
Yes. There's a separate vsmanproj for those, so that we can update the externals manifest without having to change the core PTVS manifest.

---
In reply to: [69643035](https://github.com/Microsoft/PTVS/pull/1396#discussion_r69643035) [](ancestors = 69643035)